README.md

```diff
@@ -1 +1,152 @@
-# school_library
\ No newline at end of file
+# Oop-School-library
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Oop-school-library] <a name="about-project"></a>
+
+**[Oop-school-library]**  Is an app that a Librarian at a school university uses to create new user (students/teachers), add new books, and save records of who borrowed a given book and when.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a>https://www.ruby-lang.org/en/</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[school-library-classes]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ - `bundle install`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd `school-library`
+- `git clone https://github.com/Zuwairanajme/school-library.git`
+
+
+### Install
+
+Install this project with:
+
+- cd Oop-school-library
+- bundle install
+
+### Usage
+
+To run the project execute the following command:
+
+- [N/A]
+
+### Run tests
+
+To run tests, run the following command:
+
+- `rubocop --color`
+
+
+### Deployment
+
+You can deploy this project using:
+
+- [N/A]

```

Comment:
- [optional] Instead of N/A in your readme, if the header is not necessary, let's kindly always remove it, this way, your readme is even more descriptive :+1: